<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoManMainDialog</class>
 <widget class="QDialog" name="InfoManMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信息管理</string>
  </property>
  <widget class="QTableView" name="tableViewInfoList">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>97</y>
     <width>660</width>
     <height>468</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainReturn">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>500</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainPrev">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>20</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainNext">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>100</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainDetail">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>260</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>详细</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainDel">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>340</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>25</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>信息列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainWrite">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>180</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>写信息</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoManMainDelAll">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>420</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>全删</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>17</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>pushButtonInfoManMainReturn</zorder>
  <zorder>pushButtonInfoManMainPrev</zorder>
  <zorder>pushButtonInfoManMainNext</zorder>
  <zorder>pushButtonInfoManMainDetail</zorder>
  <zorder>pushButtonInfoManMainDel</zorder>
  <zorder>label_title</zorder>
  <zorder>pushButtonInfoManMainWrite</zorder>
  <zorder>pushButtonInfoManMainDelAll</zorder>
  <zorder>tableViewInfoList</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonInfoManMainPrev</tabstop>
  <tabstop>pushButtonInfoManMainNext</tabstop>
  <tabstop>pushButtonInfoManMainWrite</tabstop>
  <tabstop>pushButtonInfoManMainDetail</tabstop>
  <tabstop>pushButtonInfoManMainDel</tabstop>
  <tabstop>pushButtonInfoManMainDelAll</tabstop>
  <tabstop>pushButtonInfoManMainReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
